manifest.versions.json

```
@@ -112,172 +90,113 @@
-    "monitor|8.0|product-version": "8.0.6",
+    "monitor|8.0|build-version": "8.0.7",
+    "monitor|8.0|product-version": "8.0.7",
     "monitor|8.0|fixed-tag": "$(monitor|8.0|product-version)",
     "monitor|8.0|minor-tag": "8.0",
     "monitor|8|major-tag": "8",
     "monitor|8.0|base-url|main": "$(base-url|public|maintenance|main)",
     "monitor|8.0|base-url|nightly": "$(base-url|public|preview|nightly)",
-    "monitor-base|8.0|linux|x64|sha": "ca711e794823b1552f4acaaf42a0a2ace6dee59ed113305eaf556c6865a93025fb40bc85fc91e2c0480ec42989a7119a11c46d4dcafda32ee9bb3c3fe31ac5cd",
-    "monitor-base|8.0|linux|arm64|sha": "af993ac36e59affcee54c5c8c45fbe26d6989690a2d746d46bfec5919e53a603ea281aa58d0bcfc2f18abf331389f9501f2936ae18f1f8fc2c56fbfac48d2a8e",
-    "monitor-ext-azureblobstorage|8.0|linux|x64|sha": "b09e1d17551d72cf2a79e673c5623a043950d578401d5a2a01c4174ea053d14d463d219715653575040802ee281ac9570ecc746da69f838009b4328285840a3e",
-    "monitor-ext-azureblobstorage|8.0|linux|arm64|sha": "6dd99144a8e8ecf2865b2b854163fc4fb779c7b877cc57696facbbd96498a121f4bc9b95e31c7694e10956320428893529e149c1fabfc405ed65d5290acb1b7f",
-    "monitor-ext-s3storage|8.0|linux|x64|sha": "6b1da4cb485c9ae70e165784175a82c25ecf3e5721d369bb2120dcf6c15566e52f4068a2af511a9d87ec8beb179e9ca90bc3729311fd4c7a112aeb8f83287df7",
-    "monitor-ext-s3storage|8.0|linux|arm64|sha": "cf4eda380ea7734f10f59c76c0424db84d149e904e42a1608817767e725260084cdc21ffcc4be82489343028b669408e2ed04833ce89f100b59e31fa82171e0d",
-
-    "monitor|9.0|build-version": "9.0.0",
-    "monitor|9.0|product-version": "9.0.0",
+    "monitor-base|8.0|linux|x64|sha": "c5a7a88c2f288e8ce307cfe6c012bb8772249bcecee7613a6ceae1e1c6978572ad78208f6d03b3a401411a3f3ea56d128e20ee4ee5a9ccf94f111602c6652594",
+    "monitor-base|8.0|linux|arm64|sha": "9a6746d1158e87f22922956c8959b6b8ea72d567c44a2709a09c7e7a1bacf6327792124bedcdead26b035ddd912ee7acc8e10f87fdccc68b955e861dfc5da12e",
+    "monitor-ext-azureblobstorage|8.0|linux|x64|sha": "2dd977feb78c424e55ffc400a08afc66de45c11ef675ba31f9e3c8a74cb06d658638023ae19adee7af384fab3c15a6b2bcd91058f9dddaea3d78bf16422f2779",
+    "monitor-ext-azureblobstorage|8.0|linux|arm64|sha": "d0bf4c2bf5959d4b7925a2246a332c59903de50d2f91ae0e80525d5a70d37ba5f59acda9971e6b80add3c8143c2b428b32e00c3002c7a851ee10ee1bfdb8d6ad",
+    "monitor-ext-s3storage|8.0|linux|x64|sha": "e3debc9dde46bc8234a44d5ef2d1be7e5e759079bb80b942b803deeb5d64c30a047aa8dfa5eb94f00d846f919f6a05999ac82fb2dbbc896a865b10a5c37a085e",
+    "monitor-ext-s3storage|8.0|linux|arm64|sha": "42763e716154a054d338eb55c4a4f0dea801282e0f6a43d6de2a695448f4767e45b446797ca5a691c0ad48f77973285c2cc51aefe439e7ec791b8e2bfd73e724",
+
+    "monitor|9.0|build-version": "9.0.1",
+    "monitor|9.0|product-version": "9.0.1",
     "monitor|9.0|fixed-tag": "$(monitor|9.0|product-version)",
     "monitor|9.0|minor-tag": "9.0",
     "monitor|9|major-tag": "9",
     "monitor|9.0|base-url|main": "$(base-url|public|preview|main)",
```

```suggestion
    "monitor|9.0|base-url|main": "$(base-url|public|maintenance|main)",
```
This doesn't make a difference for the Dockerfile generation in the main branch because both the preview and maintenance values are the same, it should be set to `maintenance` for 9.0 in the main branch.